plugins/modules/win_nssm.ps1

```diff
@@ -38,6 +38,12 @@ $stderrFile = Get-AnsibleParam -obj $params -name "stderr_file" -type "path"
 
 $executable = Get-AnsibleParam -obj $params -name "executable" -type "path" -default "nssm.exe"
 
+$app_env = Get-AnsibleParam -obj $params -name "app_environment" -type "dict" -default @{}
```

Comment:
If you want to determine if this was set or not change it to the following

```suggestion
$app_env = Get-AnsibleParam -obj $params -name "app_environment" -type "dict"
```

If not defined then it will be `$null`. Also just an FYI the `-type "dict"` does nothing for the legacy parameter handling. Unfortunately until this module is swapped to the `Ansible.Basic` style there's not much we can do about that and just expect the user to input the data properly for now.

Reply:
Done